src/popup/pages/Paratime/index.scss

```
@@ -24,6 +24,18 @@
             font-weight: normal;
             color: #7080b5;
         }
+        .runtime-info-container {
+            display: flex;
+            justify-content: space-between;
+            flex-direction: row;
+            width: 100%;
+        }
+        .runtime-info-title {
+            color: #2c3451;
+        }
+        .runtime-info-content {
+            color: #2c3451;
+        }
```

This could just be
```suggestion
            flex-direction: row;
            width: 100%;
            color: #2c3451;
        }
```
:shrug:

---

Thanks, I had adapted some code from elsewhere in this extension. Let's go with your compact way.